Prolific/Prolific/View Controllers/Submissions View Controller/SubmissionsViewController.m

```diff
@@ -132,11 +133,22 @@ - (void)didVote:(Snippet *)snippet {
                 __strong typeof (weakSelf) strongSelf = weakSelf;
                 if (strongSelf == nil) return;
                 
-                if (error) {
-                    NSLog(@"Error updating firebase with vote: %@", error.localizedDescription);
-                } else {
+                if (!error) {
                     strongSelf.round = round;
                     strongSelf.project = project;
+                    
```

Comment:
too many nslogs being left behind in commits

Reply:
sorry, I've been using them as placeholders, but I really do need to replace all of these with an error logger!

Reply:
No worries. Just trying to get you to build the habit  :)